nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
I think we should have the `not supported` assertion in both places to detect attempts to register the same synapse model multiple times.